src/ApplicationContext/UseCases/GetInTouch/GetInTouchUseCase.php

```diff
@@ -18,13 +18,14 @@
 class GetInTouchUseCase {
 
 	private $validator;
-	private $messenger;
-	private $templateBasedMailer;
+	private $forwardingMailer;
+	private $confirmingMailer;
 
-	public function __construct( GetInTouchValidator $validator, Messenger $messenger, TemplateBasedMailer $templateMailer ) {
+	public function __construct( GetInTouchValidator $validator, TemplateBasedMailer $forwardingMailer,
+								 TemplateBasedMailer $confirmingMailer ) {
 		$this->validator = $validator;
-		$this->messenger = $messenger;
-		$this->templateBasedMailer = $templateMailer;
+		$this->forwardingMailer = $forwardingMailer;
+		$this->confirmingMailer = $confirmingMailer;
```

Comment:
The method in the factory is called `newContactConfirmationMailer`. `confirmationMailer` seems more correct to me than `confirmingMailer`. Not sure about `forwardingMailer` as I don't get what it means. What do you think?